src/CoreShop/Bundle/CoreBundle/CoreExtension/StorePrice.php

```diff
@@ -468,6 +468,11 @@ public function getFromCsvImport($importValue, $object = null, $params = [])
         $storeRepo = $this->getStoreRepository();
 
         $data = $importValue == "" ? [] : json_decode($importValue, true);
+
+        if (json_last_error() !== JSON_ERROR_NONE) {
+            throw new \RuntimeException('Error decoding Store Price JSON `' . $importValue . '`: ' . json_last_error_msg());
```

Comment:
Is `\RuntimeException` the right Exception here? I would have though `\InvalidArgumentException` would be better

Reply:
You're right. Too much copy-paste.